examples/compute/main.c

```diff
@@ -50,7 +51,7 @@ int main(
     WGPUBufferId storage_buffer = wgpu_device_create_buffer(device,
```

Comment:
Is it guaranteed that a compute shader can access a buffer which is also host visible/mappable?

Reply:
It can't access it *at the same time* as it's mappable, but that's not what we need anyway. It is guaranteed that a buffer *can* support both host mapping and any other usage flags.

Reply:
I've simplified the compute example to only use one buffer now.